load_test/data/generater/playlist_pagination_cursors.py

```
@@ -70,11 +70,11 @@ def generate_playlist_cursors(sort_by: str = 'updated_at') -> None:
         query = f"""
             SELECT DATE_FORMAT({sort_column}, '%Y-%m-%dT%H:%i:%s'), HEX(id)
             FROM playlists
-            WHERE {sort_column} >= %s AND is_deleted = FALSE
+            WHERE {sort_column} >= '{recent_date}' AND is_deleted = FALSE
             ORDER BY RAND()
             LIMIT 1200
         """
```

Ruff (0.14.14)

[error] Possible SQL injection vector through string-based query construction

(S608)